crm/customizations/base_address_lau.rst

```diff
@@ -0,0 +1,72 @@
+===========================================
+Local Administrative Units (LAU) in Address
+===========================================
+
+Specify more detailed addresses for partners and companies.
+
+Configuration
+=============
+
+For this feature to work, module :code:`base_address_lau` must be
+installed and *LAU 1 in Address* and/or *LAU 2 in Address* in Business
+Documents section of General Settings must be enabled.
+
+.. image:: media/base_address_lau_conf.png
+    :align: center
+
+.. note::
+    Module :code:`base_address_lau_menus` will be installed
+    automatically to display corresponding menu items.
+
+.. tip::
+    If there is a need to load data with the main counties (largest
+    unit) and municipalities (LAU 1 middle unit) of Lithuania, module
+    :code:`l10n_lt_administrative_units` should be installed.
+
+    Example of data that will be loaded:
+
+.. image:: media/base_address_lau1_tree.png
+    :align: center
+
+LAU 1 & LAU 2
```

Comment:
Shouldnt this be added, when we fix discussed issues with module? So would not need to update documentation or something?

Reply:
Even after some changes in modules, documentation will be correct, because there is nothing said what wouldn't fit later (after changes will be made).